functions/commonfn.php

```diff
@@ -357,6 +361,33 @@ function amuRegisterUser(){
 		//return message
 		return $newid;
 	}
+
+    /**
+     * Contacts central auth to determine if uun exists
+     *
+     * @param string $uun
+     *
+     * @return boolean
+     */
+    function is_valid_uun( $uun ) {
+        global $amu_ldap_config;
+        $ldapconn  = ldap_connect(  $amu_ldap_config['ldaphost'],  $amu_ldap_config['ldapport'] ) or wp_die( "Could not connect to {$amu_ldap_config['ldaphost']}", 200 );
+        $filter    = '(&(uid=' . $uun . '))';
+        $justthese = array( '*' );
+        $sr        = ldap_search( $ldapconn,  $amu_ldap_config['dn'], $filter, $justthese );
+
+        // display entries from ldap.
+        $info = ldap_get_entries( $ldapconn, $sr );
+
+        foreach ( $info as $elt ) {
```

Comment:
Please can we change `$elt` to be a more expressive variable name. Thanks!

Reply:
changed to entry

Reply:
Thanks @richeduni - sorry but I realised that we should probably rename the `$info` variable as well. Because the function is called `ldap_get_entries()` I think the function should be renamed to `$entries` or `$ldap_entries` or something similar. Hope this is ok. Cheers!

Reply:
done

Reply:
Cheers! Looks good.